src/main/java/com/github/cameltooling/dap/internal/CamelDebugAdapterServer.java

```diff
@@ -148,18 +147,20 @@ private SetBreakpointsResponse setBreakpointsSync(SetBreakpointsArguments setBre
 						breakpoint.setNodeId(nodeId);
 						connectionManager.updateBreakpointsWithSources(breakpoint);
 						breakpointIds.add(nodeId);
-						if (sourceBreakpoint.getCondition() != null) {
-							connectionManager.getBacklogDebugger().addConditionalBreakpoint(nodeId, CAMEL_LANGUAGE_SIMPLE, sourceBreakpoint.getCondition());
-						} else {
-							connectionManager.getBacklogDebugger().addBreakpoint(nodeId);
-						}
+						addBreakpoint(sourceBreakpoint, nodeId);
 						breakpoint.setVerified(true);
+					} else {
+						breakpoint.setMessage("The Camel debugger cannot find the related id for "+ source.getPath() + " l." + line);
 					}
 				} catch (Exception e) {
-					LOGGER.warn("Cannot find related id for "+ source.getPath() + " l." + line, e);
+					String baseMessage = "The Camel debugger cannot find the related id for "+ source.getPath() + " l." + line;
+					breakpoint.setMessage(baseMessage + ". See logs for more details on exception that occured: "+ e.getMessage());
+					LOGGER.warn(baseMessage, e);
 				}
 			} else {
-				LOGGER.warn("No active routes find in Camel context. Consequently, cannot set breakpoint for {} l.{}", source.getPath(), line);
+				String message = "No active routes found in Camel context. Consequently, the Camel debugger cannot set breakpoint for "+source.getPath()+" l."+line;
```

Comment:
The string literals should be in constants or functions.

As these have some values been used, is better to put a closure or function that returns this string